app/controllers/api/v1/npq_applications_controller.rb

```diff
@@ -33,7 +33,7 @@ def reject
       end
 
       def accept
-        service = ::NPQ::Application::Accept.new(npq_application:)
+        service = ::NPQ::Application::Accept.new({ npq_application: }.merge(accept_npq_application_params["attributes"] || {}))
```

Comment:
Im not sure I remember what we agreed, is this only for v3, or for all versions? as the release note is stating it's only for v3 🤔